app/assets/stylesheets/global/_navbar.scss

```diff
@@ -18,6 +18,11 @@ nav.navbar-light {
     }
   }
 
+  // Overriding bootstrap navbar prefs causing RTL bug
```

Comment:
Can you add a bit more detail here and perhaps a URL reference, if you found this online? Are you certain this doesn't effect the navbar behavior for LTR users?

Reply:
I didn't found this online, i have read about the property and did the appropriate fix, both RTL and LTR versions are working good.

Reply:
Cool.